cpp/tests/test_odeseir.cpp

```diff
@@ -225,12 +225,33 @@ TEST(TestSeir, get_reproduction_numbers)
     mio::TimeSeries<ScalarType>::Vector result_6(4);
 
     result_0[(Eigen::Index)mio::oseir::InfectionState::Susceptible] = 9700;
+    result_0[(Eigen::Index)mio::oseir::InfectionState::Exposed] = 100;
```

Comment:
add a few comments in the test, what you are doing and why you choose these values :)